spec/system/admin/site_customization/pages_spec.rb

```diff
@@ -1,6 +1,12 @@
 require "rails_helper"
 
 describe "Admin custom pages", :admin do
+  I18n.available_locales = [:ar, :bg, :bs, :ca, :cs, :da, :de, :el, :en, :es, :"es-PE", :eu, :fa, :fr,
+                                  :gl, :he, :hr, :id, :it, :ka, :nl, :oc, :pl, :"pt-BR",
+                                  :ro, :ru, :sl, :sq, :so, :sr, :sv, :tr, :val, :"zh-CN", :"zh-TW"]
+  SiteCustomization::Page.destroy_all
+  Rails.application.load_seed
```

Comment:
Added these to make sure the test is using currently generated translations.

Reply:
@jakrzus Here's the updated patch

```diff
diff --git a/spec/lib/tasks/seed_spec.rb b/spec/lib/tasks/seed_spec.rb
index 7e64f6943e..80aa337667 100644
--- a/spec/lib/tasks/seed_spec.rb
+++ b/spec/lib/tasks/seed_spec.rb
@@ -2,18 +2,24 @@ require "rails_helper"
 
 describe "rake db:seed" do
   let(:run_rake_task) { Rake.application.invoke_task("db:seed") }
-  I18n.available_locales = [:ar, :bg, :bs, :ca, :cs, :da, :de, :el, :en, :es, :"es-PE", :eu, :fa, :fr,
-                                  :gl, :he, :hr, :id, :it, :ka, :nl, :oc, :pl, :"pt-BR",
-                                  :ro, :ru, :sl, :sq, :so, :sr, :sv, :tr, :val, :"zh-CN", :"zh-TW"]
 
   it "generates all custom pages translations populated by db:seeds" do
+    default_locales = I18n.available_locales
+
+    begin
+      I18n.available_locales = [:ar, :bg, :bs, :ca, :cs, :da, :de, :el, :en, :es, :"es-PE", :eu, :fa, :fr,
+                                :gl, :he, :hr, :id, :it, :ka, :nl, :oc, :pl, :"pt-BR",
+                                :ro, :ru, :sl, :sq, :so, :sr, :sv, :tr, :val, :"zh-CN", :"zh-TW"]
+
       SiteCustomization::Page.destroy_all
       run_rake_task
+
       paths = { accessibility: "pages.accessibility.title", conditions: "pages.conditions.title",
                 faq: "pages.help.faq.page.title", privacy: "pages.privacy.title",
                 welcome_not_verified: "welcome.welcome.title",
                 welcome_level_two_verified: "welcome.welcome.title",
                 welcome_level_three_verified: "welcome.welcome.title" }
+
       I18n.available_locales.each do |locale|
         I18n.locale = locale
         paths.each do |slug, path|
@@ -21,5 +27,8 @@ describe "rake db:seed" do
           expect(site.title).to eq I18n.t(path)
         end
       end
+    ensure
+      I18n.available_locales = default_locales
     end
+  end
 end
```